packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroupOption/ChoiceGroupOption.base.tsx

```diff
@@ -157,11 +166,50 @@ export class ChoiceGroupOptionBase extends BaseComponent<IChoiceGroupOptionProps
     );
   }
 
+  private _onLabelWrapperRef(element: HTMLDivElement | null): void {
+    if (element) {
+      const labelWrapperElement = createRef<HTMLDivElement>();
+      labelWrapperElement.current = element;
+      this._labelWrapperElementList.push(labelWrapperElement.current);
+    }
+  }
+
+  private _calculateLabelTextOverflow = (): void => {
+    if (!this._labelWrapperElementList) {
+      return;
+    }
+    for (let labelListIndex = 0; labelListIndex < this._labelWrapperElementList.length; labelListIndex++) {
+      const labelElement = this._labelWrapperElementList[labelListIndex];
+      if (labelElement && hasOverflow(labelElement)) {
+        const node = document.createElement('span');
+        const textNode = document.createTextNode('...');
+        node.appendChild(textNode);
+        this._classNames.labelOverflowWrapper && node.classList.add(this._classNames.labelOverflowWrapper);
+        labelElement.insertAdjacentElement('afterend', node);
+      }
+    }
+  }
+
   private _onRenderLabel = (props: IChoiceGroupOptionProps): JSX.Element => {
+    const imageSize = props.imageSize;
+    let gapSpace = 60;
```

Comment:
More magic numbers in the code.

Reply:
Please add some comments around as to why this number is chosen and the importance of it.

---
In reply to: [192602045](https://github.com/OfficeDev/office-ui-fabric-react/pull/4837#discussion_r192602045) [](ancestors = 192602045)